ui/components/component-library/button-base/button-base.stories.tsx

```diff
@@ -208,7 +212,7 @@ export const Rtl = (args) => (
   </Box>
 );
 
-export const Ellipsis = (args) => (
+export const Ellipsis: ComponentStory<typeof ButtonBase> = (args) => (
   <Box backgroundColor={Color.iconMuted} style={{ width: 150 }}>
```

Comment:
Ahh I see why you added `Color` to the box types. This instance here is incorrect.
```suggestion
  <Box backgroundColor={BackgroundColor.primaryMuted} style={{ width: 150 }}>
```